applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_conn.c

```diff
@@ -195,6 +261,43 @@ static void scan_recv_cb(const struct bt_le_scan_recv_info *info, struct net_buf
 	}
 }
 
+static void conn_in_set_check(struct bt_conn *conn, void *data)
+{
+	int ret;
+	struct bt_conn_info info;
+	const struct bt_csip_set_coordinator_set_member *member;
+
+	uint8_t *num_filled = (uint8_t *)data;
+
+	ret = bt_conn_get_info(conn, &info);
+	if (ret) {
+		LOG_ERR("Failed to get conn info for %p: %d", (void *)conn, ret);
+		return;
+	}
+
+	/* Don't care about connection not in a connected state */
```

Comment:
Why?

Reply:
Because we stop scanning when connection to a device, so we know that the only valid states that we care about is `CONNECTED`. Also if we find a device in `DISCONNECTED` state it shouldn't count as a set member.